### PR TITLE
Remove support for old combattag

### PR DESCRIPTION
**Pull request by BlackXnt**
@rourke750

### CODE REVIEW

File: src/vg/civcraft/mc/prisonpearl/listener/CombatTagListener.java [hunk 1]
```diff
@@ -77,27 +74,4 @@ public void handleNpcDespawn(UUID plruuid, Location loc) {
...
-        UUID plruuid = event.getPlayerUUID();
-        NPC npc = event.getNpc();
-        Location loc = npc.getEntity().getLocation();
-
-        handleNpcDespawn(plruuid, loc);
-    }
-    
-    @EventHandler
-    public void onNpcDespawnPlus(net.minelink.ctplus.event.NpcDespawnEvent event){
-    	net.minelink.ctplus.event.NpcDespawnReason reason = event.getDespawnReason();
```
Comment (Maxopoly) [inline src/vg/civcraft/mc/prisonpearl/listener/CombatTagListener.java]: Dont remove this one, this is CT+

**Comment by BlackXnt** [comment]
@Maxopoly Try to do "ok to test"

**Comment by Maxopoly** [comment]
ok to test

**Comment by Maxopoly** [comment]
implying the bot ever listens to me

**Comment by BlackXnt** [comment]
Eh I think it should be fine. I had a wrong version of CT+ in my local pom that didnt have the changes rourke made to CT+. So when I saw the red errors after removing the old combat tag imports I thought the listener was from the old combattag.

**Comment by rourke750** [comment]
Good to merge?

**Comment by BlackXnt** [comment]
yes

**Pull request merged**